bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -671,7 +680,11 @@ data:
 
     echo "" >> /opt/bitnami/redis/etc/replica.conf
     echo "replica-announce-port $REDISPORT" >> /opt/bitnami/redis/etc/replica.conf
+    {{- if .Values.replica.announceContainerIP }}
```

Comment:
`replica.announceContainerIP` is not defined in values.yaml file, but is it really needed? I think we should have a global variable for this porpoise to avoid mixing IP addresses and names